docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -67,14 +67,14 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
         'button_size' => [
           'default' => 'button_medium',
         ],
-        'card_image_size' => [
-          'default' => 'card_image_large',
+        'media_size' => [
+          'default' => 'media_size_large',
         ],
         'card_media_position' => [
-          'default' => 'card_media_position_stacked',
+          'empty_label' => t('Stacked (default)'),
```

Comment:
I think this introduced two defaults for Articles listblock <img width="146" alt="Screen Shot 2023-01-23 at 11 49 48 AM" src="https://user-images.githubusercontent.com/4663676/214115577-810058e8-dabd-4f80-8f86-7346c7a25731.png">

Reply:
In the process of thinking through a fix for this.

Reply:
I believe this is fixed now.